libft/tests/strings/ft_split_skip_delim_test.c

```diff
@@ -0,0 +1,134 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_split_skip_delim_test.c                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/23 16:24:51 by taretiuk          #+#    #+#             */
+/*   Updated: 2024/11/02 19:10:44 by taretiuk         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../sources/strings/strings.h"
+#include "tests.h"
+
+#define NUM_TEST_CASES 12
+#define MAX_ARGS 16
+
+typedef struct s_string
+{
+	char	*str;
+}	t_string;
+
+typedef struct s_string_array
+{
+	t_string	*strs;
+	int			error;
+	size_t		count;
+}	t_strings;
+
+static t_strings	create_string_array()
+{
+	t_strings	str_array;
+	str_array.count = 12;
+	str_array.error = 0;
+	str_array.strs = ft_calloc_if(sizeof(t_string) * str_array.count, 1);
+	if (str_array.strs == NULL)
```

Comment:
assert not null

Reply:
ok, then i deleted assigning error to 0, cause assert will stop execution anyway. and i dont use and dont check this flag here.